packages/react-vapor/src/components/checkbox/CheckboxActions.ts

```diff
@@ -35,8 +46,17 @@ export const removeCheckbox = (id: string): IReduxAction<ICheckboxActionPayload>
     },
 });
 
+export const disableCheckbox = (id: string, disabled: boolean): IReduxAction<ICheckboxActionPayload> => ({

```

Comment:
this should not be exported directly (only in `CheckboxReduxActions.disableCheckbox`)

Reply:
np, but why the other ones are exported directly?